script.js

```diff
@@ -1,4 +1,56 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myOutputValue = myOutputValue;
+  return inputVsGen(input);
 };
+
```

Comment:
var myOutputValue = myOutputValue; did nothing.

In your main function , maybe you can add how your game logic flow?
- computer choose  SPS
-compare input to computer SPS